### DISCUSSION TITLE
Per-listener prometheus metrics

### DISCUSSION THREAD

**Discussion by kaizensparc** [emqx/emqx#12551] [Q&A] [answered]
Hi!

I currently have a setup where I split my MQTT clients in different listeners based on their category (on-field hardware, customer API, etc.), this lets me have different authentication methods and specific ACLs. Now, I would like to make a grafana dashboard where I can see the connection_count/packet_count/etc. for each listener.

Is this possible?

Thanks :)

**Comment by thalesmg** [comment, marked as answer]
Hi.  Currently, those metrics are not labeled by listener, unfortunately.  This would have to be implemented.

CC @wivwiv
